src/structure/MasterInterfacePage.tsx

```diff
@@ -87,6 +87,12 @@ function MasterInterfacePage() {
 
   const { showMetamaskPushSnap } = useContext(AppContext);
 
+  const rewardsPagePaths = [
+    APP_PATHS.Rewards,
```

Comment:
/rewards should take user to /rewards/dashboard

Reply:
I am not sure even tat even there should be a /rewards route in first place, its upto you to figure out. But user should be take to /dashboard if he visits /rewards.